v2/internal/frontend/desktop/linux/window.go

```diff
@@ -228,6 +228,21 @@ GtkWidget* setupWebview(void* contentManager, GtkWindow* window, int hideWindowO
 
 	WebKitSettings *settings = webkit_web_view_get_settings(WEBKIT_WEB_VIEW(webview));
 	webkit_settings_set_user_agent_with_application_details(settings, "wails.io", "");
+
+	if(gpuPolicy < 0 || gpuPolicy > 2) {
+		gpuPolicy = 1; // fall back to on_demand when an unknown policy is set
```

Comment:
Should we add this as the `default` case?

Reply:
You mean that make the "on demand" policy the default?
We could do that, I set the "always" as a default because for me on popOS 22.04 and on a raspberry GTK sets it to "always" by default.

Reply:
I meant instead of this if and set gpuPolicy we could also add a default case to the switch which does set on_demand.

Reply:
Ahh yes... I don't know why I did it this way. That would be cleaner. I'll changit.

Reply:
Changed and committed.